src/mgr/Mgr.cc

```diff
@@ -588,16 +588,16 @@ bool Mgr::ms_dispatch2(const ref_t<Message>& m)
       handle_mgr_digest(ref_cast<MMgrDigest>(m));
       break;
     case CEPH_MSG_MON_MAP:
+      /* MonClient passthrough of MonMap to us */
+      handle_mon_map(); /* use monc's monmap */
       py_module_registry->notify_all("mon_map", "");
```

Comment:
Now this looks far more intuitive / less weird. 

> The call to `handle_fs_map` has always been that way. John actually mentioned a race possibility in the comment in that method:

OK, so this peculiar order has not been brought by any _fix commit_.